File: ios/XiriGoEcommerce/Feature/Home/Domain/Model/HomeBanner.swift
import Foundation

// MARK: - HomeBanner

/// A single hero banner in the home screen carousel.
struct HomeBanner: Identifiable, Equatable, Sendable {
    let id: String
    let title: String
    let subtitle: String
    let imageUrl: String?
    let tag: String?
    let actionProductId: String?
    let actionCategoryId: String?
}

File: android/app/src/main/java/com/xirigo/ecommerce/feature/home/domain/model/HomeBanner.kt
package com.xirigo.ecommerce.feature.home.domain.model

data class HomeBanner(
    val id: String,
    val title: String,
    val subtitle: String,
    val imageUrl: String?,
    val tag: String?,
    val actionProductId: String?,
    val actionCategoryId: String?,
)
